docs/medical-api/getting-started/quickstart.mdx

```diff
@@ -241,7 +242,21 @@ Data Query](/medical-api/api-reference/fhir/consolidated-data-query-post) endpoi
 We also provide the ability to add any FHIR resources to a Patient with our [Create Patient's Consolidated
 Data](/medical-api/api-reference/fhir/create-patient-consolidated) endpoint.
 
-## 9. Share Medical Data with the Networks 🚀🚀🚀
+## 9. Generate Patient Medical Record Summary ⭐⭐⭐
+
+The aforementioned FHIR resources are generated from all available CDAs for a patient, resulting in a set of unique medical records. To condense all of these records into a single document, you can follow these steps:
+
+- kick off a [Consolidated Data Query](/medical-api/api-reference/fhir/consolidated-data-query-post).
+
+This route allows you to choose what FHIR resources and what date range you want to include into the summary. When this query completes, a webhook with a JSON payload containing all these resources will be sent to you.
+
+- monitor the status of the query using the [Get Consolidated Data Query Status](/medical-api/api-reference/fhir/consolidated-data-query-get) endpoint.
+
+- check when the Medical Record Summary was created in either PDF or HTML format using the [Get Medical Record Summary Status](/medical-api/api-reference/fhir/get-medical-record-summary-status) endpoint.
+
+- download the Medical Record Summary in a PDF or HTML format using the [Get Medical Record Summary](/medical-api/api-reference/fhir/get-medical-record-summary) endpoint.
```

Comment:
I find this flow confusing. In order to generate an MR, the customer first needs to make a consolidated query? Doesn't the consolidated query endpoint contain within it the option to generate a medical record summary already by passing in the `conversionType` field. High level question here also I guess, which is why do we even need the Medical Record Summary endpoint if we already support that option in Consolidated Query?

Reply:
Good question. 

MR Summary generation is one of the core features right now, and a lot of CX use it. Currently, it's hidden behind a parameter in a function that doesn't talk much about the MR Summaries. This should bring it to light. 

As for the flow, this is why I wanted to get some high level feedback prior to getting reviews - https://metriport.slack.com/archives/C04DMKE9DME/p1706126482255269?thread_ts=1706108414.205299&cid=C04DMKE9DME

Reply:
Ok having a dedicated endpoint makes sense, but yah the flow still sounds convoluted

Reply:
I think the flow makes sense now. 

In order to generate the MR Summary, you need to use the start consolidated query. The result of that func is a webhook containing the URL to the MR summary. But this is done async and might take a while. 

The two routes I'm adding into the SDK are mainly there to support the dashboard, where one simply checks if there's already an existing MR for a patient, and another one that allows the user to download it. They are not really a part of "generating the MR summary"